.changeset/red-chefs-travel.md

```diff
@@ -0,0 +1,5 @@
+---
+'@backstage/integration': patch
```

Comment:
Snuck this in because I realized it was missing and wanna git this in todays release :grin: